### PR TITLE
[MB-8344] Show alert on home screen after saving amending orders

### PR DESCRIPTION
**Pull request by mboren**
## Description

After a service member uploads files for amended orders, and until the TOO approves the new orders, the service member should see an alert on the home screen to reassure them that they've successfully submitted the new documents.

## Reviewer Notes

I'm piggybacking on @kylehilltruss's [branch](https://github.com/transcom/mymove/pull/6906) , in which he creates some help text that displays in the same scenario that this alert is displayed in.

The formatting doesn't quite match the designs; there's a bigger gap between the text and the top of the alert box than there should be. I'm guessing this is something to do with my use of `<p>` tags. I'm still fiddling with this, but any suggestions are welcome.

Should I move the alert into its own little component in a separate file, like the Helpers are?

Are the tests adequate?

I used a `data-testid` to select the alert because I don't think I have anything else to query on that would distinguish it from other `<Alert>`s we may put on the page in the future. If this is not the best way to do this, let me know.

## Setup

We can't see this in the app yet (devseed data and the ability to upload amended orders are both in progress right now). However, you can see it in storybook:
http://localhost:6006/?path=/story/customer-components-pages-home--amended-orders

and compare against the designs:
- [mobile](https://zpl.io/V4NRGqy)
- [desktop](https://zpl.io/VKd7B5A)

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8344) for this change

### mobile
image

### tablet
image

### CODE REVIEW

**Review by kylehilltruss — APPROVED** [review]
Looks good by me 👍 

(Edit: Want to run this by someone from @transcom/truss-design?)

File: src/pages/MyMove/Home/index.jsx [hunk 1]
```diff
@@ -130,6 +130,21 @@ export class Home extends Component {
     return 'Set up your shipments';
   }
 
+  renderAlert = () => {
+    if (this.hasUnapprovedAmendedOrders) {
+      return (
+        <Alert type="success" slim data-testid="unapproved-amended-orders-alert">
+          <p>
```
Comment (Shkeating) [inline src/pages/MyMove/Home/index.jsx:137]: take the p element out of here, its adding extra space at the top and bottom
image

**Review by Shkeating — CHANGES_REQUESTED** [review]

File: src/pages/MyMove/Home/index.jsx (hunk 1 above)
Comment (Shkeating) [reply src/pages/MyMove/Home/index.jsx:137]: for the second l'd leave it so you get the correct amount of space between the paragraphs

Comment (mboren) [reply src/pages/MyMove/Home/index.jsx]: amazing. i tried every other conceivable permutation of `<p>` and `<br/>` except that one. it works!
image

**Review by Shkeating — APPROVED** [review]
beautiful!

chef's kiss

**Pull request merged**